mlflow/tracing/utils/token.py

```diff
@@ -0,0 +1,20 @@
+import contextvars
+from dataclasses import dataclass
+from typing import Optional
+
+from mlflow.entities import LiveSpan
+
+
+@dataclass
+class SpanWithToken:
+    """
+    A utility container to hold an MLflow span and its corresponding OpenTelemetry token.
+
+    The token is a special object that is generated when setting a span as active within
+    the Open Telemetry span context. This token is required when inactivate the span i.e.
+    detaching the span from the context. This will only be used when MlflowLangchainTracer
+    is configured to set the span as active span by setting `set_span_in_context=True`.
```

Comment:
Do we need to mention MlflowLangchainTracer here?

Reply:
Ah we don't need this mention, thanks for the catch!